### ISSUE TITLE
load_models now ignores old sasview 3 custom models (Trac #751)

### ISSUE THREAD

**Issue by RichardHeenan** [SasView/sasview#867]
in edit_custom_models, plugin models renamed "_v2" if  do load_models

Migrated from http://trac.sasview.org/ticket/751
```json
{
    "status": "closed",
    "changetime": "2016-10-10T19:57:28",
    "_ts": "2016-10-10 19:57:28.361888+00:00",
    "description": "in edit_custom_models, plugin models renamed \"_v2\" if  do load_models",
    "reporter": "richardh",
    "cc": "",
    "resolution": "fixed",
    "workpackage": "SasView Bug Fixing",
    "time": "2016-10-10T14:07:45",
    "component": "SasView",
    "summary": "load_models now ignores old sasview 3 custom models",
    "priority": "minor",
    "keywords": "",
    "milestone": "SasView 4.1.0",
    "owner": "mathieu",
    "type": "defect"
}
```

**Comment by RichardHeenan** [comment]
Trac update at `2016/10/10 17:40:45`: 
* **richardh** commented:


new version of "load models" now ignores sasview 3 type models


* **richardh** changed status from "new" to "assigned"
* **richardh** changed summary from "plugin models renamed "_v2" if  do load_models" to "load_models now ignores old sasview 3 custom models"

**Comment by RichardHeenan** [comment]
Trac update at `2016/10/10 17:48:41`: **richardh** changed attachment from "" to "sum_p1_p2.py"

**Comment by RichardHeenan** [comment]
Trac update at `2016/10/10 17:48:53`: **richardh** changed attachment from "" to "polynomial5.py"

**Comment by mdoucet** [comment]
Trac update at `2016/10/10 18:14:29`: 
* **mathieu** commented:

In changeset 20a70bcb7ac59e39aa57ec364463499362fa7a21:

```text
#!CommitTicketReference repository="sasmodels" revision="20a70bcb7ac59e39aa57ec364463499362fa7a21"
Make sure old-style v3 models load, and reload. Fixes #867
```


* **mathieu** changed resolution from "" to "fixed"
* **mathieu** changed status from "assigned" to "closed"

**Comment by mdoucet** [comment]
Trac update at `2016/10/10 19:57:28`: **mathieu** commented:

In changeset e4bf2716edb44b17350d7a010ad0b396cbb6472c:

```text
#!CommitTicketReference repository="sasmodels" revision="e4bf2716edb44b17350d7a010ad0b396cbb6472c"
Make sure plug-ins work with multiplication model. Re #867
```
